src/main/java/org/example/objects/Music.java

```diff
@@ -1,22 +1,43 @@
 package org.example.objects;
 
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import org.example.fuzzy_matcher.domain.Document;
+import org.example.fuzzy_matcher.domain.Element;
+import org.example.fuzzy_matcher.domain.ElementType;
+
+
 import org.example.fuzzy_matcher.domain.Document;
 import org.example.fuzzy_matcher.domain.Element;
 import org.example.fuzzy_matcher.domain.ElementType;
 
+
 import java.util.List;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class Music {
   private int id;
```

Comment:
I have a question, is the id music in this attribute loaded from the json? The id is important for Match Service